src/model/scripts/pytorch_lighting_training/pl_configurations/config.yaml

```diff
@@ -0,0 +1,10 @@
+defaults:
+  name: llama-hf-7b-from-pretrained
+  - model: llama-hf-7b-from-pretrained
+  - optimizer: adamw_optimizer
+  - scheduler: warmup_and_decay_lr
+  - trainer: default_trainer
+  - deepspeed: stage_2
```

Comment:
What should we do when there are other sharding methods e.g.. `FSDP`?

Reply:
I think we can handle these later.